crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -927,13 +932,14 @@ impl Analyzer<'_, '_> {
         }
 
         let is_symbol = normalized_types.iter().any(|ty| ty.is_symbol());
-        let is_str = normalized_types.iter().any(|ty| ty.is_str());
+        let is_str = normalized_types.iter().any(|ty| ty.is_str() || ty.is_str_lit());
```

Comment:
Please revert this line. `is_str`is not a check for `string`. It's a check for string or string literal

Reply:
but, string & "asdf" is string, number & "asdf" is never
It is right action in intersection

Reply:
I mean `ty.is_str() || ty.is_str_lit()` is same as `ty.is_str()`.
Look at the code of function.

Reply:
I fixed it